tsup.config.ts

```diff
@@ -0,0 +1,25 @@
+import type { Options } from 'tsup';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'tsup';
+
+import tsconfig from './tsconfig.json';
+
+export default defineConfig((options: Options) => ({
+  ...options,
+  entry: ['src/index.ts'],
```

Comment:
not your fault here since we don't really document but this well, but we rely on a few transpiled files from this library over in `rdme` ([example](https://github.com/readmeio/rdme/blob/51a80867c45de15e2b41af0c4bd5bbc61b932804/src/cmds/openapi/convert.ts#L2)). i tried checking out this branch locally and `npm link`-ing but wasn't able to get this working (even with the changes below) so this will require some deeper investigation 😕 is there a way to get `tsup` to transpile without bundling?

```suggestion
  entry: ['src/index.ts', 'src/rmoas.types.ts', 'src/analyzer/index.ts', 'src/lib/reducer.ts'],
```

Reply:
Will add the operation.ts to the list too, saw it someware